src/pages/studyView/addChartButton/genericAssaySelection/GenericAssaySelection.tsx

```diff
@@ -85,20 +118,30 @@ export default class GenericAssaySelection extends React.Component<
     }
 
     @computed
-    private get entityNotSelected() {
-        return _.isEmpty(this._selectedGenericAssayEntityIds);
+    private get buttonDisabled() {
```

Comment:
maybe put a comment explaining why we only disable button in studyview and not results view

Reply:
this is the kind of knowledge that another developer cannot figure out